src/styles/index.scss

```diff
@@ -1,3 +1,234 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
+
+$dark: #060b35;
+$blue: #00acdc;
+$gray: #616070;
+$light-gray: #f3f3f3;
+$white: #fff;
+
+@mixin star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+@mixin button-buy {
+  width: 100%;
+  height: 40px;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  font-weight: 700;
+  text-decoration: none;
+  font-size: 14px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo-container {
+    display: flex;
+    align-items: center;
+  }
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__list {
+    list-style: none;
+    display: flex;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    margin-left: 20px;
+    position: relative;
+    text-decoration: none;
+    text-transform: uppercase;
+    align-items: center;
+    display: flex;
+    height: 60px;
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    font-size: 12px;
+    color: $dark;
+
+    &:hover {
+      color: $blue;
+    }
+
+    &--is-active {
+      color: $blue;
+
+      &::after {
+        content: '';
+        position: absolute;
+        left: 0;
+        bottom: 0;
+        width: 100%;
+        height: 4px;
+        background-color: $blue;
+        border-radius: 8px;
+        transform-origin: bottom left;
+      }
+    }
+  }
+
+  &__list-item {
+    &:first-child .header__link {
+      margin-left: 0;
+    }
+  }
+}
+
+.product-card {
+  width: 200px;
+  border: 1px solid $light-gray;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $gray;
+    margin-top: 4px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $dark;
+  }
+
+  @for $i from 1 through 5 {
+    &__star--#{$i} {
+      background-image: url('../images/star.svg');
+
+      @include star;
+
+      &--filled {
+        background-image: url('../images/star-active.svg');
+
+        @include star;
+      }
+    }
```

Comment:
The `@for` loop generates classes for stars, but it might be more efficient to use a single class for all stars and an additional class for filled ones, to avoid unnecessary repetition in your CSS.